tests/rest/test_extension_rest_handler.py

```diff
@@ -0,0 +1,29 @@
+#
+# Copyright OpenSearch Contributors
+# SPDX-License-Identifier: Apache-2.0
+#
+# The OpenSearch Contributors require contributions made to
+# this file be licensed under the Apache-2.0 license or a
+# compatible open source license.
+#
+
+import unittest
+
+from opensearch_sdk_py.rest.extension_rest_handler import ExtensionRestHandler
+from opensearch_sdk_py.rest.extension_rest_request import ExtensionRestRequest
+from opensearch_sdk_py.rest.extension_rest_response import ExtensionRestResponse
+from opensearch_sdk_py.rest.rest_status import RestStatus
+
+
+class TestExtensionRestHandler(unittest.TestCase):
+    def test_default_handler(self) -> None:
+        handler = DefaultRestHandler()
+        self.assertEqual(len(handler.routes), 0)
+
+
+class DefaultRestHandler(ExtensionRestHandler):
+    def __init__(self) -> None:
```

Comment:
Default constructor can be removed. I also generally put helper classes as subclasses, in this case `TestExtensionRestHandler` to avoid polluting globals.

Reply:
Not sure I understand this comment.  The whole purpose of this test is to cover the default empty list for the `ExtensionRestHandler` abstract class method, so it needs to subclass that abstract class.

In Java I'd handle this as an anonymous subclass.

Reply:
You're somehow thinking that my comment is something smart ;) 

https://github.com/opensearch-project/opensearch-sdk-py/pull/48